src/interfaces/projectInterfaces.ts

```diff
@@ -148,26 +148,26 @@ export interface IProjectRequestObject {
 }
 
 export enum ProjectType {
-  ProjectComplex = 'projectComplex',
-  Street = 'street',
-  Traffic = 'traffic',
-  Sports = 'sports',
-  Omastadi = 'omaStadi',
-  ProjectArea = 'projectArea',
-  Park = 'park',
```

Comment:
Don't use uuid:s because they are environment based which means uuid:s are different in different environments

Reply:
Resolved